bindings/Python/py11File.cpp

```diff
@@ -216,7 +216,7 @@ pybind11::array File::Read(const std::string &name, const Dims &start,
             m_Stream->Read<std::string>(name, blockID).front();
         pybind11::array_t<char> pyArray(Dims{value.size()});
         std::copy(value.begin(), value.end(), pyArray.mutable_data());
-        return pyArray;
+        return std::move(pyArray);
```

Comment:
shouldn't C++11 return value optimization guarantee this already?

Reply:
you are right, however, still, not explicitly using std::move brought warnings in some of the builds